rest-api/src/main/java/io/quarkus/qe/HelloCatalog.java

```diff
@@ -0,0 +1,17 @@
+package io.quarkus.qe;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+@Path("/catalog")
+public class HelloCatalog {
+
+    @GET
+    @Produces(MediaType.TEXT_PLAIN)
+    @Path("/")
+    public String getRandomJoke() {
+        return "hello Catalog";
```

Comment:
I can't find the joke! I guess it's a copy/paste leftover, can you rename the method?